handel/src/cli/index.ts

```diff
@@ -47,8 +48,44 @@ function logCaughtError(msg: string, err: Error) {
     }
 }
 
-function logFinalResult(lifecycleName: string, envResults: EnvironmentResult[]): void {
+async function makeSureDeploymentsLogTableExists() {
+    const handelDeploymentLogsTableName = 'handel-deployment-logs';
+    const dynamoTable = await dynamodbCalls.getDynamoTable(handelDeploymentLogsTableName);
+    if (dynamoTable === null) {
+        await dynamodbCalls.createDynamoTable({
+            AttributeDefinitions: [
+                {
+                    AttributeName: 'AppName',
+                    AttributeType: 'S'
+                },
+                {
+                    AttributeName: 'EnvAction',
+                    AttributeType: 'S'
+                }
+            ],
+            KeySchema: [
+                {
+                    AttributeName: 'AppName',
+                    KeyType: 'HASH'
+                },
+                {
+                    AttributeName: 'EnvAction',
+                    KeyType: 'RANGE'
+                }
+            ],
+            ProvisionedThroughput: {
+                ReadCapacityUnits: 5,
+                WriteCapacityUnits: 5
+            },
+            TableName: handelDeploymentLogsTableName
+        });
+    }
+}
+
+async function logFinalResult(lifecycleName: string, envResults: EnvironmentResult[], handelFile: HandelFile, environmentsToDeploy: string[]): Promise<void> {
     let success = true;
+    const handelDeploymentLogsTableName = 'handel-deployment-logs';
```

Comment:
This variable value is used twice in two different variables, we should probably make it a constant value in the dynamodb module that can be referenced in both functions.